usecases/schema/tenant.go

```diff
@@ -260,24 +280,72 @@ func IsLocalActiveTenant(phys *sharding.Physical, localNode string) bool {
 		phys.Status == models.TenantActivityStatusHOT
 }
 
-func (h *Handler) getTenantsByNames(class string, names []string) ([]*models.Tenant, error) {
+func (h *Handler) getTenantsByNames(class string, names []string, after *string, limit *int64) ([]*models.Tenant, error) {
```

Comment:
Reminder we may want to revisit this once we've decided on how the API should work for all parameter combintations